micrometer-meter-provider/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ dependencies {
   api("io.opentelemetry:opentelemetry-api")
   api("io.opentelemetry:opentelemetry-sdk-metrics")
 
-  compileOnly("io.micrometer:micrometer-core:1.1.0") // do not update this version
+  compileOnly("io.micrometer:micrometer-core:1.1.0") // do not auto-update this version
```

Comment:
Is there any reason we're targeting 1.1 here? Micrometer 1.1 is super old, and not compatible with the more recent releases - the API stabilized around 1.5, and since then all the changes have been strictly incremental. Before that there's been a few breaking changes.

Reply:
makes sense, I've opened #476 to track and will send follow-up PR to make sure @HaloFour has a chance to review

Reply:
That's fine by me, I was concerned about targeting a version that would support a critical mass of users.